docs/guides/node-operators.md

```diff
@@ -0,0 +1,260 @@
+After being registered in the node operator registry, the node operator will call the `stake` function using Lido for Polygon 
+protocol (as described below) and put the desired amount of tokens to stake (10 MATIC is minimum) + allocate the amount 
+for heimdall fees (minimum 20 MATIC).
+
+## Tutorial on How to join as an Operator during testnet phase
+
+You can join as a Node Operator with an unstaked Validator node or with a pre-staked one.
+
+### Unstaked (fresh) validator
+
+If you are joining with an unstaked one, all you have to do is provide us with your 64 byte public key used on Heimdall,
+an address which will own the Operator (all the validator rewards will be sent there) and an optional name for the 
+Operator. We will create a new Operator node from the given data, but it still needs to be staked for a Validator share 
+contract to be created. When Lido for Polygon production version launches, you will be required to submit a governance proposal in 
+order to become Lido for Polygon node operator.
+
+Validator share contract is important because all the delegators will be delegating their funds to it. You will need to 
+provide 10 Matic for amount to stake and 20 more for Heimdall fee. To create it, you need to approve the total amount of 
+Matic tokens and call the function stake from the address that owns the Operator. 
+
+### Guide:
+1. Calling the stake Function
+
+   Visit https://remix.ethereum.org/, and create a new contract with an arbitrary name.
+   
+   ![xd2imfoj79nwf4aa0eml](https://user-images.githubusercontent.com/17001801/153889832-90ed824e-f08f-4916-a77c-c59fc3245b3d.png)
+   
+   Paste the following code in the created contract:
+   
+   ```solidity
+   // SPDX-FileCopyrightText: 2021 Shardlabs
+   // SPDX-License-Identifier: GPL-3.0
+   pragma solidity 0.8.7;
+   
+   interface INodeOperatorRegistry {
+       function stake(uint256 _amount, uint256 _heimdallFee) external;
+   } 
+   ```
+   Press CTRL + S to compile the code. 
+
+   After that, click on Deploy & Run transactions and make sure that `INodeOperatorRegistry` is selected in the 
+   CONTRACT drop down list. Also, make sure that Injected Web3 is selected in the ENVIRONMENT drop down list and that 
+   ACCOUNT corresponds to the address that owns the Operator (you can change the connected account via MetaMask).
+   
+   ![mgxkvjfkmjyys1mfsqi9](https://user-images.githubusercontent.com/17001801/153895814-77ba1f37-ee0d-4f35-828c-a9faf7e5dcf7.png)
+
+   Enter the address of the `NODE_OPERATOR_REGISTRY_PROXY` (please refer to the address in the deployment address table ) in the field 
+    like from the image below, and then click on the blue 
+   `At Address` button. This will provide you the interface to communicate with the NodeOperator.
+   
+   ![fdunad10cz2y2fpnxp2o](https://user-images.githubusercontent.com/17001801/153897300-ffce2e6f-1fe3-47e9-b79e-80b2bfa60897.png)
```

Comment:
New images size is not normalized.
Please, could you reduce their size? so all the images will have the same size :)